source/basic.tex

```diff
@@ -1166,7 +1166,7 @@
 that is a member of namespace \tcode{N} (where, only for the purpose of
 exposition, \tcode{N} could represent the global scope) shall be
 declared before its use in the block in which it is used or in one of
-its enclosing blocks~(\ref{stmt.block}) or, shall be declared before its
+its enclosing blocks~(\ref{stmt.block}), or shall be declared before its
```

Comment:
Seems like this comma should just be deleted. Otherwise you would also need another comma below:

> its use in namespace `N`, or, if ..., shall be declared...